<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DuelWindow</class>
 <widget class="QDialog" name="DuelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duel</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/wizardPokerLogo.png</normaloff>img/wizardPokerLogo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>1151</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ennemyBoard"/>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>1151</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ownBoard"/>
  </widget>
  <widget class="QPushButton" name="jouerButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>630</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(173, 173, 173)</string>
   </property>
   <property name="text">
    <string>Jouer une carte</string>
   </property>
  </widget>
  <widget class="QPushButton" name="attaquerButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>680</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(173, 173, 173)</string>
   </property>
   <property name="text">
    <string>Attaquer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chatButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>730</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(173, 173, 173)</string>
   </property>
   <property name="text">
    <string>Chat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finTourButton">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>650</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(173, 173, 173)</string>
   </property>
   <property name="text">
    <string>Fin de tour</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abandonButton">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>700</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(173, 173, 173)</string>
   </property>
   <property name="text">
    <string>Abandonner</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>751</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ownHand"/>
  </widget>
  <widget class="QLabel" name="ennemyName">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(173, 173, 173)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ownName">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>780</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(173, 173, 173)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ownHp">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>780</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(173, 173, 173)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ownName_3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>780</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline; color:#e60000;&quot;&gt;Hp :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ownName_4">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>780</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline; color:#e60000;&quot;&gt;Energy :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ownEnergy">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>780</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(173, 173, 173)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ennemyHp">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(173, 173, 173)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ownName_5">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline; color:#e60000;&quot;&gt;Energy :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ownName_6">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline; color:#e60000;&quot;&gt;Hp :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ennemyEnergy">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(173, 173, 173)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>741</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:600; text-decoration: underline; color:#2bd8ff;&quot;&gt;Mes cartes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="turn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>170</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(173, 173, 173)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>280</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/img/vs_logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="pub">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>594</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777213</width>
     <height>70</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
